packages/nx/src/utils/project-graph-utils.ts

```diff
@@ -29,6 +29,14 @@ export function projectHasTargetAndConfiguration(
   );
 }
 
+export function projectHasTag(project: ProjectGraphProjectNode, tag: string) {
+  return !!(
+    project.data &&
+    project.data.tags &&
+    project.data.tags.includes(tag)
+  );
```

Comment:
```suggestion
  return !!project?.data?.tags?.includes(tag);
```

Reply:
Happy to bring the change ! This was done only to make the code style the same as for `projectHasTarget` and `projectHasTargetAndConfiguration` in the same file 😅

Reply:
I don't think it's a problem if you'll even fix the existing code, it's only for the better :)

Reply:
That's now done in 9dafbd4, for other other methods too 😉